conjureup/log.py

```diff
@@ -27,24 +27,20 @@ def warning(self, msg):
 
 
 def setup_logging(app, logfile, debug=False):
-    try:
-        cmdslog = TimedRotatingFileHandler(logfile,
-                                           when='D',
-                                           interval=1,
-                                           backupCount=7)
-    except FileNotFoundError as e:
-        # SNAP: This failing
-        # May need to do something different for snappy
-        cmdslog = logging.FileHandler('deploy-done.log')
+    cmdslog = TimedRotatingFileHandler(logfile,
+                                       when='D',
+                                       interval=1,
+                                       backupCount=7)
```

Comment:
The snappy wrapper was overriding $HOME once I fixed that this worked like before